spec/02-integration/13-vaults/07-resurrect_spec.lua

```diff
@@ -232,5 +242,134 @@ describe("vault resurrect_ttl and rotation (#" .. strategy .. ") #" .. vault.nam
 end)
 
 
+describe("#multiworker vault resurrect_ttl and rotation (#" .. strategy .. ") #" .. vault.name, function()
```

Comment:
```suggestion
describe("vault resurrect_ttl and rotation (#" .. strategy .. ") #" .. vault.name, function()
```

Seems it is for local test purpose?

Reply:
No, its a word with its meaning